.travis.yml

```diff
@@ -15,5 +15,18 @@ before_script:
 
 script:
   - mdbook build
-  # See if there are changes (which means it needs to be rebuilt by the committer.)
-  - git diff-index --quiet HEAD -- || echo "Please run 'mdbook build' before pushing."
+
+branches:
+  only:
+    - master
```

Comment:
Can we skip `branches: ...`?

Reply:
While in theory we could, I am hesitant to do so since the `$GITHUB_TOKEN` env variable in Travis has a deploy key attached to my account for this repo, and I'd prefer someone didn't obtain it through a malicious PR.